doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -255,6 +255,10 @@ Libraries for networking
     * Removed ``MODEM_INFO_NETWORK_MODE_MAX_SIZE``.
     * Removed ``CONFIG_MODEM_INFO_ADD_BOARD``.
 
+* :ref:`lib_download_client` library:
+
+  * Fixed race condition when starting download
```

Comment:
```suggestion
  * Fixed a race condition when starting the download.
```

Reply:
The PR needs a rebase.

Reply:
I think this was missed :).

Reply:
@chmielewskiandreas This was not updated, so I have unresolved the conversation

Reply:
but now :) thx